kong/cache/init.lua

```diff
@@ -150,16 +150,49 @@ function _M.new(opts)
 end
 
 
+local function cache_accessed(hit_lvl)
```

Comment:
`cache_accessed` this function is only used by the statsd plugin, and should be placed in `kong.cache`